bisdn/installer/install.sh

```diff
@@ -100,11 +121,11 @@ delete_bisdn_linux_gpt_partition()
     partprobe $blk_dev
 }
 
-# Creates a new partition for the BISDN Linux OS.
+# Creates a new gpt partition for the BISDN Linux OS.
```

Comment:
we should also update line 145 to echo that we create a gpt partition

Reply:
Done.